src/components/Map/ClusterIcon.js

```diff
@@ -4,56 +4,93 @@ import L from 'leaflet';
 import * as React from 'react';
 import ReactDOM from 'react-dom';
 import { color as d3Color } from 'd3-color';
-import { interpolateWheelchairAccessibilityColors } from '../../lib/colors';
+import { interpolateWheelchairAccessibility } from '../../lib/colors';
 import isSamePlace from './isSamePlace';
 import getIconNameForProperties from './getIconNameForProperties';
-import * as categoryIcons from '../icons/categories';
+import * as markers from '../icons/markers';
+import Icon, { StyledIconContainer } from '../Icon';
 
 export default class ClusterIcon extends L.Icon {
   constructor(options: typeof L.Icon.options) {
+    // increased tap region for icons, rendered size might differ
```

Comment:
-1: Have you tested that this does not accidentally cover PoIs below clusters and makes them unreachable via tap?

Reply:
The same effect was in place before, just in Map.css. Clustering radius takes the increased click area into account, and we never had complaints so far.